spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -99,16 +99,14 @@
         grades_grade_entry_form_path(grade_entry_form_with_data, locale: 'en'))
     end
 
-    # this test is currently failing.
-    # issue #2078 has been opened to resolve this
-    # it 'does not accept a csv file with wrong data columns' do
-    #  post :csv_upload, id: grade_entry_form,
-    #       upload: { :grades_file => @file_non_empty_first_cell }
-    # expect(response.status).to eq(302)
-    # expect(flash[:error]).to_not be_empty
-    # expect(response).to redirect_to(
-    #   grades_grade_entry_form_path(grade_entry_form, locale: 'en'))
-    # end
+    it 'does not accept a csv file corrupt line endings' do
+      post :csv_upload, id: grade_entry_form,
+           upload: { :grades_file => @file_bad_endofline }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.